Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -109,10 +109,10 @@ def pull
       elsif (api_match = arg.match HOMEBREW_PULL_API_REGEX)
         _, user, repo, issue = *api_match
         url = "https://github.com/#{user}/#{repo}/pull/#{issue}"
-        tap = Tap.fetch(user, repo) if repo.start_with?("homebrew-")
+        tap = Tap.fetch(user, repo) if repo.start_with?("homebrew-", "linuxbrew-")
```

Comment:
These could probably be constants now, perhaps

```
OFFICIAL_REPO_PREFIXES = ["homebrew","linuxbrew"]
OFFICIAL_REPO_PREFIXES_DASHED = OFFICIAL_REPO_PREFIXES.map {|r| r + '-'}
```

or something like that.

It makes both this line and the line below read a little better and single-sources the list.